test/SlowTests/Client/Attachments/BulkInsertAttachments.cs

```diff
@@ -72,7 +72,7 @@ public async Task StoreManyAttachments(Options options, int count, int size)
             }
         }
 
-        [RavenTheory(RavenTestCategory.BulkInsert)]
+        [RavenRetryTheory(RavenTestCategory.BulkInsert)]
```

Comment:
now you can use the RavenRetryAttribute that was added here: 
https://github.com/ravendb/ravendb/pull/18805/files#diff-eb13f572705e88e0f143f6c17fe47a586ef7ba0b4a9ed828157d69fc3066dbacR8

Reply:
Done